### ISSUE TITLE
perl6 xt/aspell.t doc/Type/Str.pod6 # failure

### ISSUE THREAD

**Issue by coke** [perl6/doc#1359]
Need to add 'defg' to xt/codes.pws

```
$ perl6 xt/aspell.t doc/Type/Str.pod6
1..1
@(#) International Ispell Version 3.1.20 (but really Aspell 0.60.6.1)
# & defg 8 37: defog, def, deg, efg, defy, Deng, defn, deft
not ok 1 - doc/Type/Str.pod6 has 1 spelling errors

# Failed test 'doc/Type/Str.pod6 has 1 spelling errors'
# at xt/aspell.t line 68
# Looks like you failed 1 test of 1
```

**Comment by coke** [comment]
(added in 4f2611be5)

**Comment by coke** [comment]
fixed in 791f2675
